.github/workflows/gemini-pr-review.yml

```
@@ -1,456 +1,243 @@
+      - name: Gather PR Information
+        id: pr_info
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
+          # Get PR Body
+          PR_BODY=$(gh pr view ${PR_NUMBER} --json body --jq .body)
+          echo 'pr_body<<EOF' >> $GITHUB_OUTPUT
+          echo "$PR_BODY" >> $GITHUB_OUTPUT
+          echo 'EOF' >> $GITHUB_OUTPUT
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)